nvtabular/dask/io.py

```diff
@@ -172,7 +172,7 @@ def __init__(
         storage_options=None,
         **kwargs,
     ):
-
+        self.kwargs = kwargs
```

Comment:
is this used?

Reply:
yes for passing to the GPU dataset iterator.